CMakeLists.txt

```diff
@@ -80,12 +80,12 @@ endif ()
 #in the utility libraries, that don't affect the data format itself.
 # For more information see http://semver.org/
 set(ISMRMRD_VERSION_MAJOR 1)
-set(ISMRMRD_VERSION_MINOR 7)
+set(ISMRMRD_VERSION_MINOR 8)
 set(ISMRMRD_VERSION_PATCH 1)
```

Comment:
We have conventionally reset the patch number to 0 when the minor number is bumped.